### PR TITLE
#166352049 Add search functionality to meal items

### PR DESCRIPTION
**Pull request by azupatrick0**
#### What does this PR do?
It adds search functionality to meal items
#### Description of Task to be completed?
- design search input
- add functionality to enable searching meal items
#### How should this be manually tested?
- Log into the application.
- Access the Admin Panel, navigate to the Manage Menus > Meals.
- Search for a meal.
#### What are the relevant pivotal tracker stories?
- [#166352049](https://www.pivotaltracker.com/story/show/166352049)

#### Background Context
Previously, it was not possible to search for a meal
#### Screenshots (If Applicable)
searchfooditems

### CODE REVIEW

**Comment by sulenchy** [comment]
@azupatrick0 Please fix the codeclimate issue

**Comment by azupatrick0** [comment]
> Good work but the functionality takes capitalization into consideration for example if there is a meal call `Yam`, `yam` won't return any results

Noted

**Comment by azupatrick0** [comment]
> @azupatrick0 Please fix the codeclimate issue
Requires changes to other 4 files which are just 


> @azupatrick0 Please fix the codeclimate issue

This can't be fixed

**Comment by azupatrick0** [comment]
> Screen Shot 2019-06-26 at 11 51 25 AM
> 
> @azupatrick0 I don't think the styles on the search input box is in line with the existing design. You can style it to look the screenshot attached. The blue outline is very essential. You can mention me in the comment section to alert me that the change has been implemented.

Done @sulenchy

File: client/package.json [hunk 1]
```diff
@@ -9,9 +9,10 @@
     "build": "npm run prebuild && ./node_modules/.bin/webpack --config ./webpack.dev.js",
     "build:production": "npm run prebuild && ./node_modules/.bin/webpack --config ./webpack.prod.js",
     "start": "node index.js",
+    "start:dev": "npm run build && npm run dev",
     "prestart": "NODE_ENV=production npm run build:production",
     "test": "jest --runInBand --config jest.config.json  --updateSnapshot --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "test-dev": "jest --config=./jest.config.json --updateSnapshot",
+    "test-dev": "jest --config=./jest.config.json --updateSnapshot --coverage",
```
Comment (sulenchy) [inline client/package.json]: This is cool

**Comment by azupatrick0** [comment]
> Good @azupatrick0. However, I feel the search input box can look better. Look at the screenshot attach, I was able to achieve that with the style below
> 
> ```
> .search-bar {
>         color: $color-blue;
>         width: 70%;
>         // border-radius: px-to-rem(4px);
>         // padding-left: px-to-rem(4px);
>         font-size: px-to-rem(20px);
>         font-family: 'Open Sans', $font-family-pro;
>         // border: 0;
> 
>         padding: 12px 20px;
>         margin: 8px 10px;
>         box-sizing: border-box;
>         border: 2px solid #3359DF;
>         border-radius: 4px;
>       }
> ```
> 
> Screen Shot 2019-07-08 at 5 40 34 PM

Will try to re-implement it with this, thanks

**Comment by AmosWels** [comment]
> Good work but the functionality takes capitalization into consideration for example if there is a meal call `Yam`, `yam` won't return any results

> This is the fix that has been worked on in the backend. It's what I was pushing for. To have a where-like kind of search and not exact search

**Pull request merged**